src/physics.rs

```diff
@@ -12,7 +12,7 @@ impl Plugin for PhysicsPlugin {
                 gravity: Vect::ZERO,
                 ..Default::default()
             })
-            .add_plugins(RapierDebugRenderPlugin::default())
+            .add_plugins(RapierDebugRenderPlugin::default().disabled())
```

Comment:
why are we disabling here?

Reply:
with them enabled fps tanks and the screen is unreadable because the amount of krill lol. Don't know how to ignore the krill when using the debug mode. You can always scale down the number of krill and enable the plugin by removing the call to disable.